### PR TITLE
Doc 4234 - Fixes to 4.9 upgrade procedure docs

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
Fixes to 4.9 upgrade procedure docs based on feedback from SBI via @yangbor3

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/4.9/enterprise/node-upgrade-notes.md [hunk 1]
```diff
@@ -291,57 +269,56 @@ The value of `myLegalName` must exactly match the node name used in the database
...
-## Step 4: Replace `corda.jar` with the new version
-
-Replace the `corda.jar` with the latest version of Corda.
+Replace corda.jar with its latest version.
 
 Download the latest version of Corda from [our Artifactory site](https://software.r3.com/artifactory/webapp/#/artifacts/browse/simple/General/corda/net/corda/corda-node).
-Make sure it’s available on your path, and that you’ve read the [Corda release notes](release-notes-enterprise.md). Pay particular attention to which version of Java the
-node requires.
-
+Make sure it’s available on your path, and that you’ve read the [Corda release notes](release-notes-enterprise.md). Pay particular attention to which version of Java the node requires.
```
Comment (colin-hillis) [inline content/en/platform/corda/4.9/enterprise/node-upgrade-notes.md:281]: ...that the node requires.

File: content/en/platform/corda/4.9/enterprise/node-upgrade-notes.md [hunk 2]
```diff
@@ -291,57 +269,56 @@ The value of `myLegalName` must exactly match the node name used in the database
...
 
 {{< important >}}
 Corda 4 requires Java 8u171 or any higher Java 8 patch level. Java 9+ is not currently supported.
+{{< /important >}}
 
-
+{{< important >}}
+If you used Corda Shell in the previous version, put Corda Shell in the driver directory or use the standalone type Corda Shell according to Node shell.”
```
Comment (colin-hillis) [inline content/en/platform/corda/4.9/enterprise/node-upgrade-notes.md:288]: directory, or

(Also a stray " at end...???)

**Review by colin-hillis — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.9/enterprise/node-upgrade-notes.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/enterprise/node-upgrade-notes.md]: Done.

File: content/en/platform/corda/4.9/enterprise/node-upgrade-notes.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/enterprise/node-upgrade-notes.md]: Done.

**Review by yangbor3 — APPROVED** [review]

**Review by colin-hillis — APPROVED** [review]
LGTM. Thanks Paul

**Review by yarivus — COMMENTED** [review]
Thx @paulmoloneyr3

**Pull request merged**